Formula/atlas.rb

```diff
@@ -20,9 +20,11 @@ class Atlas < Formula
   def install
     ldflags = %W[
       -s -w
-      -X ariga.io/atlas/cmd/atlascmd.version=v#{version}
+      -X ariga.io/atlas/cmd/atlas/internal/cmdapi.version=v#{version}
     ]
-    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/atlas"
+    cd "./cmd/atlas" do
+      system "go", "build", *std_go_args(ldflags: ldflags)
+    end
```

Comment:
Does this really make a difference?

Reply:
TBH, I was thinking the same, but the old style did not build

Reply:
```
go
build
-trimpath
-o=/opt/homebrew/Cellar/atlas/0.5.0/bin/atlas
-ldflags=-s -w -X ariga.io/atlas/cmd/atlas/internal/cmdapi.version=v0.5.0
./cmd/atlas

main module (ariga.io/atlas) does not contain package ariga.io/atlas/cmd/atlas
```